<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBar</class>
 <widget class="QWidget" name="ToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="labelSessionName">
       <property name="text">
        <string>Session name: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineSessionName">
       <property name="text">
        <string>Diagram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDiagName">
       <property name="text">
        <string>Diagram name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineDiagName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="gridButton">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Back to grid view</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="app.qrc">
       <normaloff>:/ressources/images/grid.png</normaloff>:/ressources/images/grid.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="normalButton">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="app.qrc">
       <normaloff>:/ressources/images/normal.png</normaloff>:/ressources/images/normal.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="selectButton">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select mode </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="app.qrc">
       <normaloff>:/ressources/images/select.png</normaloff>:/ressources/images/select.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addButton">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Insert mode</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="app.qrc">
       <normaloff>:/ressources/images/add.png</normaloff>:/ressources/images/add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="centerButton">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Center diagram</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="app.qrc">
       <normaloff>:/ressources/images/center_icon.png</normaloff>:/ressources/images/center_icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="removeButton">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Delete selection</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="app.qrc">
       <normaloff>:/ressources/images/trash.png</normaloff>:/ressources/images/trash.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="gridLabel">
         <property name="text">
          <string>Grid size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="gridValueLabel">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="gridResetButton">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="app.qrc">
           <normaloff>:/ressources/images/reset.png</normaloff>:/ressources/images/reset.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="gridSlider">
     <property name="minimum">
      <number>-5</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkGrid">
       <property name="text">
        <string>Grid</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkNodesOnGrid">
       <property name="text">
        <string>Nodes on grid</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="zoomLabel">
         <property name="text">
          <string>Zoom:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="zoomValueLabel">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="zoomPercentLabel">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="zoomResetButton">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="app.qrc">
           <normaloff>:/ressources/images/reset.png</normaloff>:/ressources/images/reset.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="zoomSlider">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="pageStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="rotationLabel">
         <property name="text">
          <string>Rotation:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rotationValueLabel">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>180</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="rotationResetButton">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="app.qrc">
           <normaloff>:/ressources/images/reset.png</normaloff>:/ressources/images/reset.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="rotationSlider">
     <property name="maximum">
      <number>359</number>
     </property>
     <property name="value">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>gridSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gridValueLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zoomSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>zoomValueLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>698</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkGrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkNodesOnGrid</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotationSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rotationValueLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>932</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>888</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
